hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -1099,6 +1126,23 @@ private OzoneOutputStream createOutputStream(OpenKeySession openKey,
               decrypted.getMaterial(), feInfo.getIV());
       return new OzoneOutputStream(cryptoOut);
     } else {
+      try{
+        GDPRSymmetricKey gk;
+        Map<String, String> openKeyMetadata =
+            openKey.getKeyInfo().getMetadata();
+        if(Boolean.valueOf(openKeyMetadata.get(OzoneConsts.GDPR_FLAG))){
+          gk = new GDPRSymmetricKey(
+              openKeyMetadata.get(OzoneConsts.GDPR_SECRET),
+              openKeyMetadata.get(OzoneConsts.GDPR_ALGORITHM)
+          );
+          gk.getCipher().init(Cipher.ENCRYPT_MODE, gk.getSecretKey());
```

Comment:
This call is failing in the default tests with 
`java.security.InvalidKeyException: Illegal key size or default parameters`

Reply:
This is because the environment where these tests are failing may not have the Java Cryptography Extension (JCE) Unlimited Strength Jurisdiction Policy Files for the JDK.

@anuengineer Can we make that change on Jenkins side to install the JCE jar files?

Reply:
1. Should we make the key length by default to 128 bit instead of 256? That way I don't need to Unlimited Strength. 
2. When we get this error, say illegal key length, do you want to print out this extra information, by looking at the key length. That is, if I have set it to 32 bytes, you know it is 256 bits and just inform the user.
3. We can add this to the documentation and to the documentation of this config key.

That way, the initial user experience is smooth; suppose someone does not want to download and update Java security classes, it will still work.

Reply:
@anuengineer Agreed.
1. Made default to 128 bit (16 char) so that users are not forced to have JCE policy jars.
2. Added the log message with more details in case it hits the invalidKeyException
3. We have HDDS-2059 logged, which would allow users to specify random secret, so we can tackle the documentation aspect during that work.